docs/report/report.tex

```diff
@@ -189,174 +189,164 @@ \subsection{Deployment Cycle}
 \subsubsection{Probleme}
-\colorbox{red}{TODO:}welche prrobleme sind aufgetreten?}
-
+Zu Beginn ging der Kundenwunsch, dass bereits NFC-Tags ausgelesen und angezeigt werden können, unter und führte \colorbox{yellow}{zu Problemen mit der Zeit}.
+Abgesehen davon war nur die Einarbeitung in die genutzten Technologien für manche schwieriger.
 \subsubsection{Lösungen}
-\colorbox{red}{TODO:}Welche Lösungen haben wir gefunden?}
-
+Bereits von Anfang an zeigte sich ein sehr guter Zusammehalt im Team und bei Problemen wurde sich schnell untereinander im Team geholfen.
+Um den Kundenwunsch noch sinnvoll umzusetzen, wurden gegen Ende des Sprints Überstunden geleistet.
 \subsubsection{Product Increment}
-\colorbox{red}{TODO:}Was ist am Ende dabei rumgekommen?}
-
+Das Produkt wurde hierbei sinnvoll um ein erstes User Interface erweitert, welches die Tag-ID vom angelegten NFC-Tags anzeigen konnte.
+Der NFC-Reader wurde mit dem Raspberri Pi verbunden und als ein auslieferbares Docker Image bereitgestellt.
+Auch wurde für die Entwicklung automatisierte Tests erstellt, welche neuen Code auf etablierte Code-Konventionen und Lauffähigkeit überprüften.
 \subsubsection{Retrospektive}
-\colorbox{red}{TODO:}Was haben wir dabei gelernt? Neue Erkentnisse? Neue Sichtweisen?
-Was lief gut, neu gelernt, was lief nicht so gut, was verbessern?}
-\colorbox{red}{TODO:}
-  Gut
-- Pair Programming
-- (github) workflow
-- team arbeit
-- Gegenseitige Hilfe
-- Workflow lief besser
-
-gelernt
-- Viel Technisches (Feathers, Hooks, OAuth)
-- manchmal weniger komplex denken und mehr an konkreten Problemen arbeiten
-- Latex
-
-nicht so gut
-- abhängigkeiten von tickets (zumindest augenscheinlich, manchmal ist das gar nicht so der fall)
-- Alles zu sehr gefühlt voneinander noch dependant, dass manche länger mal auf andere Sachen warten müssen
-
-Verbessern
-- Früher nachfragen
-- probleme direkter ansprechen
-- mehr pair-programming
-- Presentation besser vorbereiten und vorher durchgehen
-- in Daily: Cooolen Code vorstellen
-- Definition-of-Done erstellen
-- letzter Punkt, sollte sich aber von selber lösen
-\end{comment}
+Vor allem der Einsatz und das Know-How vom Teammitglied Anton wurde hier wertgeschätzt, welcher sich um die Automatisierungen von Tests und Deployment kümmerte.
+Auch das Gruppenklima wurde vom Team sehr gelobt, die Zusammenarbeit, Kommunikation und die regelmäßigen Meetings waren liefen sehr gut.
+Neu gelernt hat das Team bei diesem Meilenstein vor allem wie Vue, Feathers und Tailwind funktionieren, sowie kamen auch viele das erste mal Automatisierungsprozessen, Code- und Pull Request Reviews in Kontakt.
+Das Team bemängelte bei dem Meilenstein aber auch, dass der Kundenwunsch zu spät behandelt wurde und die Prioritäten falsch gelegt wurden.
```

Comment:
```suggestion
Das Team bemängelte bei dem Meilenstein aber auch, dass der Kundenwunsch zu spät behandelt wurde und die Prioritäten falsch gesetzt wurden.
```

Reply:
umgesetzt in https://github.com/project-leek/project-leek/pull/409/commits/db7fedecbe3f12f759a2c504c4b9f0e5a7061fee